docs/snippets/common/my-component-exclude-tags.story.js.mdx

```diff
@@ -0,0 +1,14 @@
+```js
+// MyComponent.stories.js|jsx
+
+import { MyComponent } from './MyComponent';
+
+export default {
+  component: MyComponent,
+  tags: ['no-tests'], // 👈 Excludes all stories from being tested with the test-runner
```

Comment:
Should the phrasing be more along the lines of "adds the 'no-tests' tag to all stories"? Because technically the tags don't do anything, they're just attached metadata, and you can then use them for different filtering e.g. `--excludeTags no-tests` but also `--includeTags no-tests`